docs/guides/end-to-end-testing/migration/selenium-to-cypress.mdx

```diff
@@ -0,0 +1,399 @@
+
+You will first [install Cypress](/guides/getting-started/installing-cypress).
+You will then [open the Cypress application](/guides/getting-started/opening-the-app).
+
+Check out our in-depth [E2E getting started guide](/guides/end-to-end-testing/writing-your-first-end-to-end-test).
+
+And now you're all set to start the migration.
+
+## Strategy and Considerations
+
+If the current automation framework supports Cucumber, jump to the section [Migrating test cases with Cucumber](#Migrating-test-cases-with-Cucumber).
+
+Before starting to write any code, take the time to assess the architecture of your framework. A well-structured
+framework will be a great deal easier to maintain and expand in the future.
+
+Some questions to discuss:
+
+- Will we use Page-objects?
+- Will we mock API calls?
+- How will we divide the tests?
+
+For example: While the utilization of the Page Object Model (POM) has been widely discussed and embraced as a best
+practice, there remains divergence in opinion regarding its implementation. Some advocate for a minimalist approach,
+wherein the POM solely comprises page selectors, while others advocate for a more comprehensive approach that includes
+both selectors and all possible page actions. It's crucial to engage in team discussions to align on POM structure,
+thereby avoiding surprises during the initial pull request.
+
+Another critical consideration is whether the team intends to leverage
+[Cypress intercept functionality](/api/commands/intercept) to stub API calls.
+While this approach reduces the need for end-to-end tests, it significantly increases the number of smaller tests.
+For some teams, the heightened maintenance overhead may outweigh the benefits. However, stubbing API requests can diminish
+backend dependencies and enhance framework stability.
+
+In larger applications, locating specific tests within the framework can prove challenging. Therefore, establishing
+guidelines for test organization is essential for creating a maintainable framework. Some teams prefer organizing tests
+based on application functionality, while others opt for organization by webpage. While performance and functionality
+remain unaffected, clear guidelines on test placement are imperative for framework maintainability.
+
+These considerations serve as a roadmap toward crafting a well-structured, easily maintainable, and enjoyable-to-use framework.
+Investing time in contemplation and extensive team discussions regarding these questions will yield significant dividends in the long run.
+
+With thorough preparation underway, it's time to initiate the actual migration process.
+
+One of the first things to migrate would be all page-objects or selectors, and depending on the frontend framework the
+developers are using, this could be mostly copy and paste. For instance, in Vue.js the selector in Selenium would be
+`vl-checkbox` while in Cypress this would be `.vl-checkbox`. Granted, the difference is annoying to fix,
```

Comment:
Still curious about this one. It almost implies the use of css selectors which we advise against as a best practice. It might make sense just as a simple reference example, I just wasn't clear on the origin.